### ISSUE TITLE
Word Break inconsistency between Chrome and Firefox

### ISSUE THREAD

**Issue by ghost** [ikrishg/reseter.css#19]
**Is your feature request related to a problem? Please describe.**
Chrome doesn't wrap its links at all while Firefox does.
Chrome uses a different default value for word-break then Firefox as after adding word-break: break-word; on its links it behaved the same way as Firefox did.
I also saw that similar questions have come up before and they were put aside because of "unexpected behaviors", but this seems like a special case as it to only happens with the <a> tag so it could be an exception.

**Describe the solution you'd like**
```css
a {
  cursor: pointer;
  color: #3333e1;
  background-color: transparent;
  text-decoration: none;
  word-break: normal;
}
```

**Additional context**
Originally Opened By @Gaboris at necolas/normalize.css#851 Plus, Original Solution By @VixieTSQ in necolas/normalize.css#852

**Comment by ghost** [comment]
Waiting For Some One To Grab This Good First Issue! Well if anyone is interested kindly comment here (I'll Assign It You And Only You)

**Comment by VixieTSQ** [comment]
> Waiting For Some One To Grab This Good First Issue! Well if anyone is interested kindly comment here (I'll Assign It You And Only You)

I already made a pull request. So... why give it off to someone else?

**Comment by ghost** [comment]
@VixieTSQ We are **fixing the issues of other related projects** to make `reseter.css` perfect. You made a pull request to [normalize.css repo](https://github.com/necolas/normalize.css) the code is different in `reseter.css` and `normalize.css`. The Things Remain The Same But Repositories And Files And Sources Change. So it means that we can't merge the pull request from the [normalize.css repo](https://github.com/necolas/normalize.css) to this repo. If you want you can [submit a pull request](https://github.com/krishdevdb/reseter.css/compare) here as well. But You Were The Original Person To Solve It. You Deserve Appreciation Thus Our All Contributors Bot Will Add You To The [Contributors List](https://github.com/krishdevdb/reseter.css#contributors). :)

**Comment by krishdevdb** [comment]
We have started to fix the issues of similar projects in this project see #16

**Comment by krishdevdb** [comment]
The orignal idea was yours @VixieTSQ so

@all-contributors please add @VixieTSQ for code

**Comment by VixieTSQ** [comment]
> @VixieTSQ We are **fixing the issues of other related projects** to make `reseter.css` perfect. You made a pull request to [normalize.css repo](https://github.com/necolas/normalize.css) the code is different in `reseter.css` and `normalize.css`. The Things Remain The Same But Repositories And Files And Sources Change. So it means that we can't merge the pull request from the [normalize.css repo](https://github.com/necolas/normalize.css) to this repo. If you want you can [submit a pull request](https://github.com/krishdevdb/reseter.css/compare) here as well. But You Were The Original Person To Solve It. You Deserve Appreciation Thus Our All Contributors Bot Will Add You To The [Contributors List](https://github.com/krishdevdb/reseter.css#contributors). :)

Oh sorry, sorry when I got this notification I thought it was on the normalize.css repository. Nevermind

**Comment by ghost** [comment]
No Problem @VixieTSQ

**Comment by ghost** [comment]
Fixed In #30
